src/app.rs

```diff
@@ -73,6 +73,21 @@ fn build_app() -> App<'static, 'static> {
                 .help("Perform exactly NUM runs for each command. If this option is not specified, \
                        hyperfine automatically determines the number of runs."),
         )
+        .arg(
+            Arg::with_name("build")
+                .long("build")
+                .short("b")
```

Comment:
Concerning the name: I guess there are other uses for this besides "building" your software. How about renaming it to `--setup`? The help text can and should still refer to the build/compile example though.

Reply:
I'm agnostic on the name, and would have preferred `-setup`, but `-s` is taken for `--style` already. Less typing than:

    hyperfine --setup '...' -p '...' '<cmd>' -c '...'

So pretty much anything as long as it's got a short name is fine by me.

Just looking at https://www.thesaurus.com/browse/set%20up `--build`, `--begin` and `--erect` would all fit and have `-b` and `-e` available, maybe there's others.

I also see from the git history that `--prepare` used to be called `--setup`, but I don't know if you've got users that expected that that would be surprised, or if it's OK to just re-use it.

Reply:
> Thinking about it a bit again I think `--begin` is a better name than

> with `build` I'd think the name immediately suggests to the user

So what are you advocating for now? :smile: `--build` or `--begin`?

I kind of like `--setup`, as it indicates that it is the opposite of `--cleanup`, which is - as far as I undertand this proposal - what the implemented new option does, right?

Reply:
Let's go with `--setup`, updated & stole `-s` from `--style`. As noted in the updated commit message it might be nice to print a warning on `-s basic|full|...`, but I couldn't find out how to do that.

Reply:
> it might be nice to print a warning on `-s basic|full|...`, but I couldn't find out how to do that.

Once `options.setup_command` is just an `Option<String>`, you can do:
```
match options.setup_command {
  Some("basic") | Some("full") | … => {
    /* fail with error */
  },
  _ => {}
}
```